functions/v2/helloHttp/index.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START region_tag_here]
```

Comment:
I think gen 2 should get the existing region tag and the gen 1 sample should be updated with a functions_gen1

Reply:
With Katie's perspective, it's probably best to keep the region tags the same, so all she has to do is update file path (this should be the same with Drift tooling).

Reply:
I think some DRIFT tooling may assume globally-unique region tags. (That's something for them to confirm.)

**If so**, we should figure out which set of tags (v1 vs. v2) should change.

----------

That said, I think there's lots of ambiguity around what our region tag approach should be. Perhaps we should build consensus on that before we move forward.

Reply:
**Update:** @averikitsch and I discussed this in a (super-mega-ultra-WIP) doc, and we both landed on **reusing existing region tags**.

(It might break some convenience features in DRIFT, but it's less docs work for @labtopia.)